src/Processor/Gotenberg.php

```diff
@@ -159,6 +159,12 @@ public function getPdfFromString(string $html, array $params = [], bool $returnF
             $chromium->extraHttpHeaders($params['extraHttpHeaders']);
         }
 
+        if (isset($params['metadata']) && method_exists($chromium, 'metadata')) {
+            $chromium->metadata($params['metadata']);
+        } elseif (isset($params['metadata'])) {
+            $chromium->formValue('metadata', json_encode($params['metadata']));
+        }
```

Comment:
I think it is enough to check the existence of metadata once.
```suggestion
        if (isset($params['metadata'])) {
            if (method_exists($chromium, 'metadata')) {
                $chromium->metadata($params['metadata']);
            } else {
                $chromium->formValue('metadata', json_encode($params['metadata']));
            }
        }
```